src/controllers/auth.js

```diff
@@ -24,7 +25,7 @@ const createDefaultTables = async () => {
   // create _users table
   if (!usersTable) {
     // create the _users table
-    tableService.createTable('_users', dbTables.userSchema);
+    tableService.createTable('_users', schema.userSchema);
```

Comment:
Then we can use it here, e.g.
```
tableService.createTable(reservedTableNames._USERS, schema.userSchema);
```

Reply:
This comment is fixed in the next PR https://github.com/thevahidal/soul/pull/160